tests/e2e/creation_test.go

```diff
@@ -353,6 +360,46 @@ func (tc *testContext) validateDSC() error {
 	return nil
 }
 
+func (tc *testContext) testTrustedCABundle() error {
+	CAConfigMapName := "odh-trusted-ca-bundle"
+	CADataFieldName := "odh-ca-bundle.crt"
+
+	if tc.testDSCI.Spec.TrustedCABundle.ManagementState == operatorv1.Managed {
+		foundConfigMap := &corev1.ConfigMap{}
+		err := tc.customClient.Get(tc.ctx, client.ObjectKey{
+			Name:      CAConfigMapName,
+			Namespace: tc.testDSCI.Spec.ApplicationsNamespace,
+		}, foundConfigMap)
+
+		if err != nil {
+			return fmt.Errorf("Config map not found, %w", err)
+		}
+
+		checkNewline := strings.HasSuffix(foundConfigMap.Data[CADataFieldName], "\n")
+
+		if checkNewline == false {
+			fmt.Print("Newline not found at the end of configmap")
```

Comment:
probably, better to use testing.T logging.

Reply:
@ykaliuta Yep this is done